backend/src/main/java/com/recipes/backend/bizz/ingredient/domain/Ingredient.java

```diff
@@ -14,6 +14,8 @@
 
     private String name;
 
+    private String quantity;
```

Comment:
String quantity?

Reply:
Yes, according to documentation we provide values such as: 10g, 2 slices, 1 cup and so on